atlp-rwanda/phoenix-bn-backend
# Ch add mocha chai and sinon

**What does this PR do?**

 - Test the welcome endpoint with mocha and chai 

**Description of Task to be completed**

- added mocha and chai for unitest and sinon for spy and mocks
- setup the test folder and test .js file for testing welcome endpoint and create a space for further tests 


 **How should this be manually tested?**

1. Cloning this repository to your local machine. 

2. Switch to a branch named ch-add-mocha-chai-andSinon

3. Open Terminal on your code editor and type `npm i` then type `npm test` to start the testing

4. after npm test you should see one test passed
 
**What is the relevant Trello story?**
- https://trello.com/c/C3AgjVc1

 **Screenshots (if appropriate)**
![test](https://user-images.githubusercontent.com/41580903/95905321-fe71e800-0d98-11eb-92f5-15a6b0c0a1f0.PNG)
